docs/protos/kvetch/api/v1/api.proto

```diff
@@ -25,28 +22,25 @@ service API {
 
   // Subscribe will subscribe to a key or prefix and return the current value
   // and any changes.
-  rpc Subscribe(stream SubscribeRequest) returns (stream SubscribeResponse);
+  rpc Subscribe(SubscribeRequest) returns (stream SubscribeResponse);
 }
 
 message SetValuesRequest { repeated KeyValue messages = 1; }
 
 message SetValuesResponse {}
 
-message GetValuesRequest { repeated string keys = 1; }
+message GetValuesRequest {
+  // GetValue is a get value request.
+  message GetValue {
```

Comment:
im noticing in the code that keys are stored as byte[] as well. might be stretching the scope to allow that right now but its cool that it potentially could be added here.

Reply:
i'm in favor of enforcing human-readable strings as keys, not because I think it'll prevent any bugs but because it will make our lives a lot easier when debugging

Reply:
i definitely agree w/that for our use-case. although, if open sourced, i could see somebody seeing a use-case for object based keys or whatever. fork away i guess